runtime/nutterfixture.py

```diff
@@ -54,6 +54,9 @@ def execute_tests(self):
         return TestExecResults(self.test_results)
 
     def __load_fixture(self):
+        if hasattr(self, 'data_loader') == False:
+            raise InitializationException("If you have an __init__ method in your test class, make sure you make a call to initialize the parent class.  For example: NutterFixture.init(self)")

```

Comment:
I actually stumbled upon this :) Does this require a call to `NutterFixture.init` or simply to the NutterFixture constructor?

Reply:
Nice catch - adding to PR